src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildAction.java

```diff
@@ -25,42 +33,39 @@
  *
  * @author Ross Rowe
  */
-public class SauceOnDemandBuildAction extends AbstractAction {
+public class SauceOnDemandBuildAction extends AbstractAction implements SimpleBuildStep.LastBuildAction {
 
     /**
      * Logger instance.
      */
     private static final Logger logger = Logger.getLogger(SauceOnDemandBuildAction.class.getName());
 
     /**
-     * String pattern for the URL which retrieves Sauce job details from the REST API.
-     */
-    private static final String JOB_DETAILS_URL = "http://saucelabs.com/rest/v1/%1$s/build/%2$s/jobs?full=true";
-
-    /**
      * Regex pattern that is used to identify Sauce job ids which have been run as part of a Jenkins build.
      */
     public static final Pattern SESSION_ID_PATTERN = Pattern.compile("SauceOnDemandSessionID=([0-9a-fA-F]+)(?:.job-name=(.*))?");
 
-    private AbstractBuild<?, ?> build;
+    private Run build;
```

Comment:
yup, I agree if variable name is tightly coupled does not make sense to break it.